src/Modal/Modal.tsx

```diff
@@ -92,14 +92,25 @@ const modalBackdrop = css`
   }
 `;
 
+const modalContents = (size: Props["size"]) => css`
+  background-color: white;
+  border-radius: 12px;
+  box-shadow: 0 16px 32px 0 rgba(0, 0, 0, 0.12),
+    0 0 0 1px rgba(18, 21, 26, 0.04);
+  max-height: 80%;
+  overflow-y: auto;
+  padding: ${size === "large" ? "40px" : "32px"};
+  width: ${getModalWidth(size)}px;
+`;
+
 /* istanbul ignore next */
 function assertUnreachable(value: never): never {
   throw new TypeError(`Unreachable value reached ${value}`);
 }
 
 type TLength = string | 0 | number;
 
-const getModalWidth = (size: Props["size"]): CSS.WidthProperty<TLength> => {
+function getModalWidth(size: Props["size"]): CSS.WidthProperty<TLength> {
```

Comment:
Q: What criteria are we using here to decide function vs arrow function? I noticed you switched this but added an arrow function above. Is this just a statement vs block statement body difference?

Reply:
I copied this from your PR 😃 It was failing eslint because this `const` was declared after it was used (`const` values aren't hoisted like functions). I should clean it up though.

Reply:
I typed a message on your PR and forgot to send it talking about this point 😃

In my opinion, we should use `function`s unless we have a reason not to. An example of when we need to use a `const` would be when we have a type that defines the arguments of a function and its return type, like `React.FC`.